tests/e2e/system_test/test_cr_resources_validation.py

```diff
@@ -72,24 +118,71 @@ def test_network_fence_not_editable(self):
 
             try:
                 run_cmd(f"sh {temp_file.name}")
-                network_fence_modified_yaml = run_oc_command(
-                    f"get {network_fence_name} -o yaml"
+                cr_resource_modified_yaml = run_oc_command(
```

Comment:
any reason we changed the name to cr_resource instead of network_fence?

Reply:
Yes. once it was a function which dealed with network fence object only. 
Now it is a general function  which gets the object name (it may be  "network fence" for one test and "crone job" for another), so all the names are general to handle different types of cr resorces